### PR TITLE
Mythril

### PR DESCRIPTION
**Pull request by MadmanMartian**
Makes Mythril forgeable

### CODE REVIEW

File: code/modules/mining/ores_coins.dm [hunk 1]
```diff
@@ -236,6 +236,12 @@
 	icon_state = "telecrystal"
 	material="telecrystal"
 
+/obj/item/weapon/ore/mythril
+	name = "mythril ore"
+	desc = "a naturally-occuring silver steel alloy."
```
Comment (DamianX) [inline code/modules/mining/ores_coins.dm]: `a` -> `A`

File: code/modules/mining/materials.dm [hunk 2]
```diff
@@ -137,6 +137,7 @@ var/global/list/initial_materials	//Stores all the matids = 0 in helping New
 	var/sharpness_mod = 1
 	var/quality_mod = 1
 	var/melt_temperature
+	var/armor_mod = 1
```
Comment (DamianX) [inline code/modules/mining/materials.dm:140]: I assume this is planned to be used in the future?

File: __DEFINES/setup.dm [hunk 3]
```diff
@@ -782,6 +782,7 @@ SEE_PIXELS	256
 #define MAT_BRASS   	"$brass"
 #define MAT_RALLOY   	"$ralloy"
 #define MAT_ICE			"$ice"
+#define MAT_MYTHRIL		"&mythrl"
```
Comment (DamianX) [inline __DEFINES/setup.dm]: the other macros are using `$` instead of `&`

File: code/modules/mining/materials.dm (hunk 2 above)
Comment (MadmanMartian) [reply code/modules/mining/materials.dm:140]: Yes

File: __DEFINES/setup.dm (hunk 3 above)
Comment (SonixApache) [reply __DEFINES/setup.dm]: shouldn't this be `$mythril` for that matter

**Comment by SGuivenne** [comment]
Does it prevent bullets from sticking in wounds?  (Note: Mythril, as per LOTR is basically immune to being pierced but allows some level of kinetic energy through, bone breaks are reasonable, shrapnel is not).

**Pull request merged**